### PR TITLE
Store sqlite test db files in test_output directory

### PR DESCRIPTION
**Pull request by clemens-msupply**
This helps to keep the project directory clean.

### CODE REVIEW

**Review by andreievg — APPROVED** [review]

File: src/util/test_db.rs [hunk 1]
```diff
@@ -75,6 +79,17 @@ pub async fn setup(db_settings: &DatabaseSettings) {
...
 
+#[cfg(feature = "postgres")]
+fn make_test_db_name(base_name: String) -> String {
+    base_name
+}
+
+#[cfg(feature = "sqlite")]
+fn make_test_db_name(base_name: String) -> String {
+    // store all test db files in a test directory
+    format!("test_output/{}", base_name)
```
Comment (wlthomson) [inline src/util/test_db.rs:90]: Could we clean-up the directory after the tests are all finished?

**Review by wlthomson — APPROVED** [review]
LGTM.

Would also prob be good to add this to `.gitignore`?

File: src/util/test_db.rs (hunk 1 above)
Comment (clemens-msupply) [reply src/util/test_db.rs:90]: Had the same thought and I am still not sure about it. I think its actually nice to keep the sqlite files for debugging, e.g. when something went wrong you can straight look into the db and you don't have to disable the deletion first... However, you could also just rerun the test and set a breakpoint at the correct line...

Comment (clemens-msupply) [reply src/util/test_db.rs:90]: *.sqlite is already in .gitignore...

**Pull request merged**